src/strings/PowerFxResources.en-US.resx

```diff
@@ -1304,6 +1304,14 @@
   <data name="AboutColorFade_fade_or_column" xml:space="preserve">
     <value>A percentage (or column of percentage values) by which the color (or column of color values) will be faded. A negative percentage produces a darker shade. A positive percentage produces a lighter shade.</value>
   </data>
+  <data name="AboutFileInfo" xml:space="preserve">
```

Comment:
>AboutFileInfo [](http://example.com/codeflow?start=14&length=13)

We probably going to need a `AboutFileInfo_file_blob` description.

Reply:
good catch - it would be nice if we had a test to catch this.